packages/web-components/src/components/markdown/markdown.ts

```diff
@@ -77,6 +77,12 @@ class DDSMarkdown extends LitElement {
       this._hasRendered = true;
       render(lightDOMTemplateResult, this, { eventContext: this });
     }
+    this.querySelectorAll('bx-unordered-list').forEach(e => {
+      e.setAttribute('isexpressive', '');
+    });
+    this.querySelectorAll('bx-ordered-list').forEach(e => {
+      e.setAttribute('isexpressive', '');
+    });
```

Comment:
These selectors can be combined. Also, even though `isexpressive` is a string I think we should stick to the camel-casing (`isExpressive`).

```suggestion
    this.querySelectorAll('bx-ordered-list, bx-unordered-list').forEach(e => {
      e.setAttribute('isExpressive', '');
    });
```